src/wagtail_personalisation/wagtail_hooks.py

```diff
@@ -125,65 +126,131 @@ def dont_show_variant(parent_page, pages, request):
     return utils.exclude_variants(pages)
 
 
-@hooks.register("register_page_listing_buttons")
-def page_listing_variant_buttons(page, page_perms, *args, **is_parent):
-    """Adds page listing buttons to personalisable pages. Shows variants for
-    the page (if any) and a 'Create a new variant' button.
+if WAGTAIL_VERSION >= (5, 2):
```

Comment:
The compatibility is different between <= 5.1 and >= 5.2
The difference is with the `classname` attr and the `user` permissions param